lib/mc_translator.rb

```diff
@@ -57,7 +57,7 @@ def project_root(path = '.')
     end
 
     def git
-      Git.open(project_root)
+      Git.open(project_root('..'))
```

Comment:
thoughts on having this read from config?

Reply:
It can be done, do we plan on using this on a different project or changing our project structure again? Just curious.